src/components/clusterexplorer/node.configurationvalue.ts

```diff
@@ -0,0 +1,26 @@
+import * as vscode from 'vscode';
+
+import { Kubectl } from '../../kubectl';
+import { Host } from '../../host';
+import { ClusterExplorerNode, ClusterExplorerNodeImpl, ClusterExplorerConfigurationValueNode } from './node';
+import { ResourceKind } from '../../kuberesources';
+
+export class ConfigurationValueNode extends ClusterExplorerNodeImpl implements ClusterExplorerConfigurationValueNode {
+    constructor(readonly configData: any, readonly key: string, readonly parentKind: ResourceKind, readonly parentName: string) {
+        super("configitem");
```

Comment:
nit: single quotes.

not-nit `configitem` string is repeated several times, extract a constant.